src/app/app.routes.ts

```diff
@@ -1,3 +1,23 @@
 import { Routes } from '@angular/router';
+import { HomeComponent } from './home/home.component';
 
-export const routes: Routes = [];
+export const routes: Routes = [
+  {
+    path: 'General-Chat',
+    children: [
+      {
+        path: '',
+        component: HomeComponent,
+      },
+      {
+        path: ':name',
+        component: HomeComponent,
+      },
+    ],
+  },
+  {
+    path: '',
+    redirectTo: '/General-Chat/General-Chat',
```

Comment:
I think same here could be lower case `general-chat`
could be without `/`

also I think it should be on top of routes config by priority